fuel-moshi/src/test/kotlin/fuel/moshi/FuelMoshiTest.kt

```diff
@@ -39,6 +39,25 @@ internal class FuelMoshiTest {
         }
     }
 
+    @Test
+    fun testReifiedTypeMoshiResponse() = runBlocking {
+        val mockWebServer = MockWebServer().apply {
+            enqueue(MockResponse().setBody("{\"userAgent\": \"Fuel\"}"))
+        }
+
+        withContext(Dispatchers.IO) {
+            mockWebServer.start()
+        }
+
+        val response = Fuel.get(mockWebServer.url("user-agent"))
```

Comment:
jscpd found too many duplicates (7.69%) over threshold (0%)

Change these variables with different name

Reply:
What is jscpd? ... 😅

Reply:
Okay fixed, maybe.